packages/design-system/src/icons/eraser.svg

```diff
@@ -1,5 +1,5 @@
 <svg width="80" height="80" viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
-  <path stroke="#131516" stroke-width="2.4707" stroke-linecap="round" d="M36.7627 57.7637h27.1777"/>
-  <path fill-rule="evenodd" clip-rule="evenodd" d="M42.5143 51.3916l18.5648-18.5648c.1725-.1724.29-.3921.3378-.6312L62.678 25.89c.1729-.8644-.5892-1.6265-1.4536-1.4536l-6.3056 1.2611c-.2391.0478-.4588.1654-.6312.3378L35.7228 44.6001l6.7915 6.7915zm-8.269 6.5219c1.4473 1.4473 3.7939 1.4474 5.2412 0l23.3396-23.3396c.5174-.5174.8701-1.1763 1.0136-1.8938l1.2611-6.3056c.5186-2.5932-1.7677-4.8795-4.3609-4.3609l-6.3056 1.2612c-.7175.1435-1.3764.4961-1.8938 1.0135L29.2009 47.6279c-1.4473 1.4473-1.4473 3.7939 0 5.2412l5.0444 5.0444z" fill="#131516"/>
-  <path d="M23.7475 34.3632c-.1971-.5325-.9503-.5325-1.1473 0l-1.6415 4.4362c-.062.1675-.194.2995-.3614.3614l-4.4362 1.6416c-.5325.197-.5325.9502 0 1.1472l4.4362 1.6415c.1674.062.2994.194.3614.3614l1.6415 4.4362c.197.5325.9502.5325 1.1473 0l1.6415-4.4362c.0619-.1674.1939-.2994.3614-.3614l4.4362-1.6415c.5325-.197.5325-.9502 0-1.1472l-4.4362-1.6416c-.1675-.0619-.2995-.1939-.3614-.3614l-1.6415-4.4362zM30.9684 24.3473c-.1314-.355-.6335-.355-.7649 0l-1.0943 2.9575c-.0413.1116-.1293.1996-.241.2409L25.9108 28.64c-.355.1314-.355.6335 0 .7649l2.9574 1.0943c.1117.0413.1997.1293.241.2409l1.0943 2.9575c.1314.355.6335.355.7649 0l1.0943-2.9575c.0413-.1116.1293-.1996.2409-.2409l2.9575-1.0943c.355-.1314.355-.6335 0-.7649l-2.9575-1.0943c-.1116-.0413-.1996-.1293-.2409-.2409l-1.0943-2.9575z" fill="#131516"/>
+  <path stroke="currentColor" stroke-width="2.4707" stroke-linecap="round" d="M36.7627 57.7637h27.1777"/>
```

Comment:
@samwhale These `currentColor` changes everywhere are fine, however our SVGO config ought to do this automatically for us. (Why) is that not working anymore?

Reply:
I'll take a look at these after the webworker stuff 👍. These three icons weren't getting updated. svgo config may need some updates

Reply:
Fixed! [862cf5e](https://github.com/GoogleForCreators/web-stories-wp/pull/5792/commits/862cf5e3d667ac15eaf912b667a288f812b0d5d6)